tests/server/TestConfig.js

```diff
@@ -59,15 +59,6 @@ for (const appName of appsList) {
       assert.equal(clientConfig.apiHost, 'https://addons.allizom.org');
       assert.equal(conf.util.getEnv('NODE_ENV'), 'stage');
     });
-
-    it(`should provide a development conf for ${appName}`, () => {
```

Comment:
> Did they exist just as a safety check against node-config regressions? Or to make sure our override configs exist?

Both. Config is a fairly critical piece so I wanted to have a safety net incase upstream did something that meant it would break us.